test/Microsoft.Extensions.ML.Tests/Microsoft.Extensions.ML.Tests.csproj

```diff
@@ -19,13 +19,13 @@
     <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="$(MicrosoftExtensionsTestPackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="$(MicrosoftExtensionsTestPackageVersion)" />
   </ItemGroup>
-    
```

Comment:
Would it be possible to revert the unnecessary changes in this PR? If we want to make these formatting changes, I think they should be a separate PR.

Reply:
Yep. I'll go ahead and revert them.

Reply:
Reverted